### PR TITLE
[ADD] hr_employee_skills_menu: Tree and form views for the object hr …

### PR DESCRIPTION
**Pull request by Berezi**
…employee skill.

### CODE REVIEW

**Comment by Berezi** [comment]
@anajuaristi @oihane @Daniel-CA @alfredoavanzosc review please.

File: hr_employee_skills_menu/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Berezi - AvanzOSC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Hr Employee Skills Menu",
+    'version': '14.0.1.0.0',
+    "author": "Avanzosc",
+    "website": "https://www.avanzosc.es",
+    "category": "Sales",
```
Comment (Daniel-CA) [inline hr_employee_skills_menu/__manifest__.py]: 'category': 'Human Resources/Employees',

File: hr_employee_skills_menu/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: hr_employee_skills_menu
+#: model:ir.ui.menu,name:hr_employee_skills_menu.hr_employee_skill_menu
+msgid "Emlpoyee Skills"
```
Comment (Daniel-CA) [inline hr_employee_skills_menu/i18n/es.po]: Employee

File: hr_employee_skills_menu/i18n/hr_employee_skills_menu.pot [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: hr_employee_skills_menu
+#: model:ir.ui.menu,name:hr_employee_skills_menu.hr_employee_skill_menu
+msgid "Emlpoyee Skills"
+msgstr ""
```
Comment (Daniel-CA) [inline hr_employee_skills_menu/i18n/hr_employee_skills_menu.pot:21]: Employee

Comment (Berezi) [reply hr_employee_skills_menu/i18n/hr_employee_skills_menu.pot:21]: Aldaketak eginda. Eskerrik asko.

**Review by anajuaristi — APPROVED** [review]

**Review by Daniel-CA — APPROVED** [review]

**Pull request merged**